### PR TITLE
Sclang: print version with -v

### PR DESCRIPTION
**Pull request by gusano**
Re #1324 but that one uses `-v` instead

### CODE REVIEW

**Comment by crucialfelix** [comment]
build failure:  https://travis-ci.org/supercollider/supercollider/jobs/62197344

```
/home/travis/build/supercollider/supercollider/lang/LangSource/SC_TerminalClient.cpp: In member function ‘void SC_TerminalClient::printUsage()’:
/home/travis/build/supercollider/supercollider/lang/LangSource/SC_TerminalClient.cpp:128:4: error: expected ‘)’ before ‘memGrowBuf’
/home/travis/build/supercollider/supercollider/lang/LangSource/SC_TerminalClient.cpp:132:3: warning: format ‘%s’ expects argument of type ‘char*’, but argument 4 has type ‘int’ [-Wformat]
/home/travis/build/supercollider/supercollider/lang/LangSource/SC_TerminalClient.cpp:132:3: warning: format ‘%d’ expects argument of type ‘int’, but argument 5 has type ‘const char*’ [-Wformat]
/home/travis/build/supercollider/supercollider/lang/LangSource/SC_TerminalClient.cpp:132:3: warning: format ‘%s’ expects a matching ‘char*’ argument [-Wformat]
make[2]: *** [lang/CMakeFiles/libsclang.dir/LangSource/SC_TerminalClient.cpp.o] Error 1
make[1]: *** [lang/CMakeFiles/libsclang.dir/all] Error 2
make: *** [all] Error 2
```

travis is teh awesome

File: lang/LangSource/SC_TerminalClient.cpp [hunk 1]
```diff
@@ -122,7 +124,7 @@ void SC_TerminalClient::printUsage()
 			"   -s                             Call Main.stop on shutdown\n"
 			"   -u <network-port-number>       Set UDP listening port (default %d)\n"
 			"   -i <ide-name>                  Specify IDE name (for enabling IDE-specific class code, default \"%s\")\n"
-			"   -a                             Standalone mode\n",
+			"   -a                             Standalone mode\n"
```
Comment (crucialfelix) [inline lang/LangSource/SC_TerminalClient.cpp]: ,

**Comment by gusano** [comment]
@crucialfelix thanks, force pushed

**Comment by gusano** [comment]
That one can be merged as well ;)

**Comment by bagong** [comment]
Haha, okle ;)

**Comment by bagong** [comment]
Wah, wrong button ;)

**Comment by bagong** [comment]
The travis failure is a git problem, not a build failure

**Pull request merged**